4D Plugin API/4DPluginAPI.c

```diff
@@ -22,6 +22,10 @@
 #include <stdlib.h>
 #include <string.h>
 
+#if __APPLE__
+#include <CoreGraphics/CoreGraphics.h>
```

Comment:
Why is this include necessary?

Reply:
in one of my projects I got an error where ``CGContextRef`` used in ``PA_UseQuartzAxis`` and ``PA_UseQuickdrawAxis `` was undefined.

Reply:
Ok, prefer using `VERSIONMAC` which is defined in Flags.h than `__APPLE__`